Specifications/Tool/com.apple.build-tools.swift-stdlib-tool.xcspec

```diff
@@ -0,0 +1,115 @@
+/**
+ Copyright (c) 2015-present, Facebook, Inc.
+ All rights reserved.
+
+ This source code is licensed under the BSD-style license found in the
+ LICENSE file in the root directory of this source tree. An additional grant
+ of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+{
+    Type = Tool;
+    Identifier = com.apple.build-tools.swift-stdlib-tool;
+    Name = "Swift stdlib tool";
+
+    /* TODO: Need swift-stdlib-tool to construct valid command line. */
+    CommandLine = "builtin-swiftStdLibTool [options]";
+    RuleName = "CopySwiftLibs $(OutputPath)";
+
+    DeeplyStatInputDirectories = YES;
+    Outputs = (
+        "$(OutputPath)",
+    );
+
+	Options = (
+        {
+            Name = "SWIFT_STDLIB_TOOL_ACTION";
+            CommandLineArgs = ( "--copy" );
+        },
+        {
+            Name = "SWIFT_STDLIB_TOOL_VERBOSITY";
+            Type = Enumeration;
+            Values = (
+                none,
+                verbose,
+            );
+            DefaultValue = "verbose";
+            CommandLineArgs = {
+                none = ();
+                verbose = (
+                    "--verbose",
```

Comment:
I'm not super familiar with xcspec syntax, so I apologize if this is already true here, but keep in mind `swift-stdlib-tool` has two verbosity settings: `--verbose` (verbose output) and `--verbose --verbose` (debugging output). Does this allow for `--verbose` to be passed twice?